client/src/cmdlfem4x50.c

```diff
@@ -244,14 +237,14 @@ static void print_info_result(PacketResponseNG *resp, const em4x50_data_t *etd,
 
     prepare_result(data, 0, 33, words);
 
-    bpwc = check_bit_in_byte(7, words[2].byte[2]);   // password check
-    braw = check_bit_in_byte(6, words[2].byte[2]);   // read after write
-    fwr = reflect8(words[2].byte[0]);    // first word read
-    lwr = reflect8(words[2].byte[1]);    // last word read
-    fwrp = reflect8(words[1].byte[0]);   // first word read protected
-    lwrp = reflect8(words[1].byte[1]);   // last word read protected
-    fwwi = reflect8(words[1].byte[2]);   // first word write inhibited
-    lwwi = reflect8(words[1].byte[3]);   // last word write inhibited
+    bpwc = (bool)(words[2].byte[2] & 128);  // password check (bit 7)
+    braw = (bool)(words[2].byte[2] & 64);   // read after write (bit 6)
```

Comment:
Usually you would use a enum or defines,  to specify what each bit does.
and compare it accordingly.  This way the code becomes much more readable and documented at the same time.
```
#define  EM4x50_CONF_BLK  2
#define  EM4x50_SETTING_BYTE  2

#define PWD_CHK  0x80
#define READ_AFTER_WRITE 0x40

if ( (words[EM4x50_CONF_BLK].byte[EM4x50_SETTING_BYTE] & PWD_CHK) == PWD_CHK) 

```

Reply:
I agree and increased readability by adding numerous defines -> done